packages/react-emotion/package.json

```diff
@@ -23,7 +23,8 @@
     "create-emotion-styled": "^9.1.2"
   },
   "peerDependencies": {
-    "emotion": "^9.1.3"
+    "emotion": "^9.1.3",
+    "react": ">= 15"
```

Comment:
Those `>=` ranges are too wide, we can't guarantee being compatible with future versions of react/preact, so it's "dangerous" to make this assumption here. If we are gonna be compatible with future versions then we should loosen up this range in the future to include next versions. 

Let's just stick to something like `15.x || 16.x` here and `8.x` for `preact`

Reply:
Allright. I think these need to be changed these as well then.
https://github.com/emotion-js/emotion/blob/d9a9e332dfbc13ea78325082393ccdd54d34afd6/packages/create-emotion-styled/package.json#L24
https://github.com/emotion-js/emotion/blob/d9a9e332dfbc13ea78325082393ccdd54d34afd6/packages/emotion-theming/package.json#L50-L51

Reply:
yeah, imho they should be too, good catch 👍